lib/ApplicationFeatures/ConfigFeature.cpp

```diff
@@ -71,6 +77,13 @@ void ConfigFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
       new BooleanParameter(&_checkConfiguration),
       arangodb::options::makeDefaultFlags(arangodb::options::Flags::Uncommon,
                                           arangodb::options::Flags::Command));
+
+  options->addOption(
```

Comment:
As this option only has an effect when the executable was built with glibc, we could as well wrap the entire option into the ifdefs here as well.

Reply:
Yes, but then the observable behaviour of the executable (does the option exist) would change more between different builds (and OSes). I would prefer to leave it this way and have the option everywhere but render it meaningless outside of Linux/glibc builds.